isminet/clients/unifi.py

```
@@ -0,0 +1,426 @@
+"""UniFi Network API client implementation."""
+
+from typing import Dict, Any, List, Type, TypeVar, cast
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)